src/renderer/src/i18n/locales/es.json

```
@@ -302,6 +302,15 @@
       }
     },
     "lib": {
+      "linear": {
```

Biome (2.5.6)

[error] The key linear was already declared.

(lint/suspicious/noDuplicateObjectKeys)